src/main/scala/cromwell/engine/backend/jes/JesConfiguration.scala

```diff
@@ -1,16 +1,26 @@
 package cromwell.engine.backend.jes
 
+import cromwell.engine.backend.jes.authentication.JesDockerCredentials
+import cromwell.engine.io.gcs.GoogleConfiguration
+import cromwell.util.ProductionDockerConfiguration
+
 /**
  * Trait for JesConfiguration
  */
 trait JesConfiguration {
   def jesConf: JesAttributes
+  def googleConf: GoogleConfiguration
+  def dockerConf: Option[JesDockerCredentials]
 }
 
 object ProductionJesConfiguration {
   lazy val jesConf =  JesAttributes()
+  lazy val googleConf =  GoogleConfiguration.gcloudConf.get
```

Comment:
`GoogleConfiguration.gcloudConf` is a `Try` wrapping the parsing of the Google configuration, which is using scalaz to validate the configuration.
So it fails an exception containing all the reasons while the config is not correct will be thrown.